pa11y/src/mappings/reservation/data/fbi/patron.json

```diff
@@ -0,0 +1,29 @@
+{
+	"authenticateStatus": "VALID",
+	"patron": {
+		"address": {
+			"coName": "",
+			"street": "Kongens Nytorv 34",
+			"postalCode": "1050",
+			"city": "København",
+			"country": "DK"
+		},
+		"allowBookings": false,
+		"birthday": "1999-01-01",
+		"blockStatus": null,
+		"defaultInterestPeriod": 180,
+		"emailAddress": "john.johnson@johnmail.com",
```

Comment:
Maybe use `example.com` which is block for testing. `johnmail.com` is currently owned by at domain shark... just to be sure to be sure... that this never ends up sending mails in anyway.